src/main/java/UI.java

```diff
@@ -0,0 +1,67 @@
+import java.util.LinkedList;
+import java.util.Scanner;
+
+public class UI {
+
+    private String dividerLine = new String("-").repeat(80).concat("\n");
+    
+    /*UI.start()
+        Prints Start Message
+        Awaits input
+        While input is not bye, pass input to parser
+    
+            format the output by parser -> implies parser should always return LinkedList of strings 
+            or throws exception
+            print it out 
+            
+            throw exception if required
+            return to awaiting input above
+        Else
+            print byebye message
+            exit.
+     */
+    
+    public void start(TaskList tasks, FileHandler fileHandler) {
+        LinkedList<String> startMessage = new LinkedList<String>();
+        startMessage.add("Hello from");
+        startMessage.add(" ____        _        ");
+        startMessage.add("|  _ \\ _   _| | _____ ");
+        startMessage.add("| | | | | | | |/ / _ \\");
+        startMessage.add("| |_| | |_| |   <  __/");
+        startMessage.add("|____/ \\__,_|_|\\_\\___|");
+        startMessage.add("Hello! I'm Duke");
+        startMessage.add("What can I do for you?");
+        
+        this.formattedPrint(startMessage);
+        
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        
+        while (!input.equals("bye")) {

```

Comment:
Ugh not sure how this works with the GUI. I could not generate the welcome message somehow. But I abandoned while loop and the program still runs on the GUI. I guess the Main/Launcher/Application class has something like that to keep it going?

Reply:
Heh, caught red-handed once again... (was rushing for the deadline).
I removed the bulk of the UI class and replaced it with static methods following your suggestion above.